README.md

```diff
@@ -38,7 +33,7 @@ In order to obtain a local copy of this repository run:
 
 ## Live Demo
 
-[Live Demo Link](https://glittering-pie-9a3643.netlify.app/)
+[Live Demo Link](https://math-magicians-eva.netlify.app/)
```

Comment:
[OPTIONAL] It looks like your uploaded screenshot is not clear. Kindly add another picture that could describe your webapp.